WordPress/gutenberg
# [RNMobile] Update picker component's .scss file to follow BEM naming convention

## Related PRs

* `gutenberg-mobile`: https://github.com/wordpress-mobile/gutenberg-mobile/pull/5075

## What?

This PR updates the Picker component's files to follow BEM naming convention, in accordance with [WordPress coding guidelines](https://developer.wordpress.org/block-editor/contributors/code/coding-guidelines/).

## Why?

Some updates were made to the component's .scss file in https://github.com/WordPress/gutenberg/pull/42996, leaving some CSS selectors that followed the proper convention and some that didn't. This PR brings more consistency by applying the correct convention to all selectors in the file.

## How?

This PR updates the name of all CSS selectors referenced throughout the Picker component's files to follow BEM naming convention.

## Testing Instructions

1. Navigate to the editor in the iOS and/or Android WordPress app.
2. To access the Picker component, add an image block and tap the option to add image. Another option for accessing the component is to tap the ellipsis in the bottom right of a block for the actions menu.
3. Confirm that this PR doesn't introduce any unexpected colour changes in the WordPress app.
